src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -144,6 +144,10 @@ export default class CheckoutPlugin extends Plugin {
             onChange: this.onPaymentMethodChange.bind(this)
         });
 
+        if (paymentMethod.type === 'scheme') {
+            configuration.enableStoreDetails = true;
```

Comment:
I'm guessing we should enable this only if the checkbox is ticked? Part of the OneClick payments ticket

Reply:
We should only enable this when the user is logged in and for now the payment method is scheme
Later we are going to add more payment methods to the list which can be tokenised.

This configuration serves the purpose to show the checkbox.

Reply:
Ah didn't see this was part of the configuration before creating the component 👍

Reply:
@cyattilakiss we have a user ID even if the user is not signed in in Shopware. So if the session lasts for long enough, the token can be reused.

Reply:
I would only let them see the checkbox if they are logged in otherwise it can be confusing that we let them store the payment method but then they are probably never able to reuse the stored payment method. Also I don't think it's secure to rely on the session in this case because there is no way for the customer to break their sessions (like logging out) and they would left their stored payment method in the session. The second part is pretty though I was just thinking loud :)